src/vendor/wordpress/_wordpress.scss

```
@@ -251,3 +251,29 @@ $wp-button-gap: size.$spacing-list-inline-gap;
   line-height: size.$height-control-multiline; /* 1 */
   padding: size.$padding-control-vertical 0;
 }
+
+/**
+ * Groups
+ * Rounds the corners for groups when the has-background class is applied.
+ * 1. This value is added to keep the Cloud Four card containers and
+ * the Gutenberg styles consistent.
+ * 2. When the alignfull class is applied the border radius will go
+ * back to 0 to keep the corners flush with the edge of the page.
+ * 3. The alignwide class will have the border-radius set to 0 until it hits
+ * the large breakpoint because at that point, it no longer sits along the
+ * edge of the screen. It's intended to be applied within a container.
+ */
+
+.wp-block-group.has-background {
+  border-radius: size.$border-radius-card-contained; /* 1 */
+}
+
+.wp-block-group.has-background.alignfull {
+  border-radius: 0; /* 2 */
+}
+
+.wp-block-group.has-background.alignwide {
+  @media (max-width: breakpoint.$l) {
```

Unfortunately this media query will take effect one pixel too late, because technically we're saying "take effect until anything _larger_ than `breakpoint.$l`."

Luckily, we have the [postcss-media-minmax](https://github.com/postcss/postcss-media-minmax) polyfill added so you should be able to revise this line like so:

```suggestion
  @media (width < breakpoint.$l) {
```